build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export DATASOURCE=github
+export LOOKUP_NAME=CocoaPods/CocoaPods
+export VERSION_SCHEME=ruby
+export START_VERSION=1.7.2
```

Comment:
Sure we shouldn't start any earlier? It's not really a big problem to build a lot

Reply:
Yes, but I worry about previous versions: `pod` will fetch full specs repo before any lockfile update. Actually it seems to be fetching also when `source 'https://github.com/artsy/Specs.git'` is still present in the newest `Podfile` and `Podfile.lock`.

Reply:
Potentially, it's solvable with some cleaning preprocessing in git tempdir, but I think that's out of scope for now.

Reply:
Do you mean you suggest only supporting the versions that use the new CDN?

Reply:
Yes, as a "good enough" solution for the full cloning problem

Reply:
Technically wouldn't the old versions still work, but just be slower? i.e. this affects the generation of lock file and not our internal logic

Reply:
Technically, yes. Just realised, for images we can actually pre-fetch them on build time. It will increase its size, but should speed up artifact generation.

Reply:
@rarkins What do you think? Let's try or not?